pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -689,3 +689,37 @@ def test_rdivmod_invalid(self):
     def test_td_op_timedelta_timedeltalike_array(self, op, arr):
         with pytest.raises(TypeError):
             op(arr, Timedelta('1D'))
+
```

Comment:
these need to go in pandas/tests/scalar/test_nat.py; need to parameterize on the op itself, and should include both Timestamp and NaT (pretty sure we have the Timestamp tests already but can't hurt)

Reply:
done